stellargraph/layer/attri2vec.py

```diff
@@ -29,6 +29,16 @@
 from ..mapper import Attri2VecLinkGenerator, Attri2VecNodeGenerator
 
 
+def _require_without_generator(value, name):
+    if value is not None:
+        return value
+    else:
+        raise ValueError(
+            f"'{name}' is None. 'input_dim', 'node_num' and 'multiplicity' must be specified when a "
```

Comment:
For the phrasing of this error message, I wonder if we can go for something more formulaic (consistency makes things easier to parse, IME), e.g.

```python
f"{name}: expected a value for 'input_dim', 'node_num' and 'multiplicity' when 'generator' is not provided, found {name}=None."
```

Reply:
Thanks, I'll do the same for https://github.com/stellargraph/stellargraph/pull/1130